pulp_ansible/app/models.py

```diff
@@ -285,3 +293,57 @@ class AnsibleDistribution(Distribution):
 
     class Meta:
         default_related_name = "%(app_label)s_%(model_name)s"
+
+
+def _gen_secret():
+    return os.urandom(32)
+
+
+class ContentRedirectContentGuard(ContentGuard):
```

Comment:
There is a redirecting content guard with preauthenticated URL in pulp_container. Can we reuse that one (after moving it to pulpcore)?
https://github.com/pulp/pulp_container/blob/main/pulp_container/app/models.py#L588

Reply:
We can. IIRC, we just copied the redirect content guard from pulp container pretty as is.

Reply:
+1 to moving this to pulpcore.